students/vvinodh/Lesson3/list_lab.py

```diff
@@ -0,0 +1,58 @@
+#Series 1
+fruits = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print (fruits)
+add_fruit = input("What fruit do you want to add? ")
+fruits.append(add_fruit)
+print (fruits)
+response = int(input("Type in a number "))
+print(response," - ",fruits[response:response+1])
```

Comment:
instead of using slicing, you can use indexing -> `fruits[response + 1]`

I think this is also causing a bug, if I type in `1` it gives me "Pears" instead of "Apples"